crowbar_framework/lib/crowbar/backup/restore.rb

```diff
@@ -163,7 +163,7 @@ def restore_chef
 
               record = JSON.load(file.read)
               filename = file.basename.to_s
-              if proposal?(filename) && type == :databags
+              if proposal?(filename) && !repo?(file) && type == :databags
```

Comment:
Sounds good, except that you don't want to negate the call to `crowbar_databag?` I think :-)